symbolic-common/src/types.rs

```diff
@@ -804,37 +782,37 @@ impl<'a> Name<'a> {
     /// # Example
     ///
     /// ```
-    /// use symbolic_common::{Name, NameMangling};
+    /// use symbolic_common::{Language, Name, NameMangling};
     ///
-    /// let unmangled = Name::new("foo::bar", NameMangling::Unmangled);
+    /// let unmangled = Name::new("foo::bar", NameMangling::Unmangled, Language::Unknown);
     /// assert_eq!(unmangled.mangling(), NameMangling::Unmangled);
     /// ```
     pub fn mangling(&self) -> NameMangling {
         self.mangling
     }
 
-    /// Converts this name into a `Cow`, dropping the language.
+    /// Converts this name into a [`Cow`].
     ///
     /// # Example
     ///
     /// ```
-    /// use symbolic_common::{Name, NameMangling};
+    /// use symbolic_common::Name;
     ///
-    /// let name = Name::new("_ZN3foo3barEv", NameMangling::Unknown);
+    /// let name = Name::from("_ZN3foo3barEv");
```

Comment:
In these examples, you know that the name is mangled. I would propose to encourage explicit usage of `NameMangling` here:

```suggestion
    /// let name = Name::new("_ZN3foo3barEv", NameMangling::Mangled, Language::Cpp);
```